dipy/workflows/reconst.py

```diff
@@ -21,6 +21,77 @@
 from dipy.reconst.shm import CsaOdfModel
 from dipy.workflows.workflow import Workflow
 
+from dipy.reconst import mapmri
+import matplotlib
+matplotlib.use('agg')
+import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+class ReconstMAPMRIFlow(Workflow):
+    @classmethod
+    def get_short_name(cls):
+        return 'mapmri'
+
+    def run(self, data_file, data_bvecs, data_bvals, small_delta, big_delta):
+        img = nib.load(data_file)
```

Comment:
Documentation is really important here. You can look at the example [here](https://github.com/nipy/dipy/blob/master/dipy/workflows/reconst.py#L36) how to make a good documentation